src/components/IntegrationTests/utils/test-utils.ts

```diff
@@ -0,0 +1,17 @@
+import { fireEvent, screen, act } from '@testing-library/react';
```

Comment:
move this file under __tests__?

Reply:
File has been moved

Reply:
It looks like the test run failed because only tests are expected under `__tests__`... would moving this back to `utils`  be fine?

Reply:
@sahil143 I reverted the change to fix the tests from failing (see above). If you have any recommendations on where else I should move this besides `__tests__` to prevent the failure again please let me know.

Reply:
My only concern is that the testing utilities should not be kept with the actual component utilities. I like the idea of creating separate utilities for opening dropdowns that can be used everywhere else. You can rename the functions to something more specific like `openIntegrationTestContextDropown` and move them to the common test-utils in src/utils/test-utils.ts. I am open to any other suggestions.